### PR TITLE
search: allow fetching of more subreddit results

### PR DESCRIPTION
**Pull request by curioussavage**
:eyeglasses: @ajacksified 

This also fixes empty searches!

### CODE REVIEW

File: src/views/pages/search.jsx [hunk 1]
```diff
@@ -107,14 +108,19 @@ class SearchPage extends React.Component {
   }
 
   handleShowMoreClick(e) {
-    var list = e.currentTarget.previousSibling;
-    list.className = list.className.replace('list-short-view', '');
+    var props = this.props;
+    var url = this._composeUrl({
+      query: props.query.q,
+      type: 'sr'
```
Comment (ajacksified) [inline src/views/pages/search.jsx]: needs comma

**Comment by ajacksified** [comment]
The "more subreddits" button never shows up.

**Comment by ajacksified** [comment]
:haircut:

**Comment by curioussavage** [comment]
:haircut:

**Comment by ajacksified** [comment]
:fish:

**Comment by curioussavage** [comment]
3696d9f90b857ad3501141cc568b7157a158dfd2

**Pull request closed without merging**